test/integration/openshift_health_checker/setup_container.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Required vars:
+# * scenario = unique name for the container to be started
+# * image = name of the image to start in the container
+# Optional vars:
+# * l_groups = host groups that the container should be added to
+# * l_host_vars = any variables that should be added to the host
+
+- name: Start specified container
+  gather_facts: no
+  hosts: localhost
+  connection: local
+  vars:
+    container_name: openshift_ansible_test_{{ scenario }}
+  tasks:
+    - name: start container
+      docker_container:
+        name: "{{ container_name }}"
+        image: "{{ image }}"
+        command: sleep 1800
+        recreate: yes
+    - name: add host
+      add_host:
+        ansible_connection: docker
+        name: "{{ container_name }}"
+        groups: '{{ l_groups | default("OSEv3,masters,nodes") }}'
```

Comment:
True, there's code early on that adds all those group members into OSEv3 so there's no reason to list it explicitly.